ci/bin/format.dart

```diff
@@ -231,7 +225,7 @@ abstract class FormatChecker {
           'failed to apply.');
       completedJobs
           .where((WorkerJob job) => job.result.exitCode != 0)
-          .map<String>((WorkerJob job) => job.result.output)
+          .map<String?>((WorkerJob job) => job.result.output)
```

Comment:
It looks like output is never null:
https://pub.dev/documentation/process_runner/latest/process_runner/ProcessRunnerResult/output.html